Inc/ST-LIB_LOW/Sensors/EncoderSensor/EncoderSensor.hpp

```diff
@@ -0,0 +1,32 @@
+
+/*
+ * AnalogSensor.hpp
+ *
+ *  Created on: Nov 9, 2022
+ *      Author: ricardo
+ */
+
+#pragma once
+#include "ST-LIB.hpp"
+#include "ST-LIB_LOW/Sensors/Sensor/Sensor.hpp"
+#define counter_distance 0.025
+#define clock_frequency 300000000
+
+
+class EncoderSensor : public Sensor<double>::Sensor{
+public:
+	EncoderSensor(Pin pin1, Pin pin2, double* position, double* speed, double* acceleration);
+	void read();
+	uint8_t getID();
```

Comment:
```suggestion
	uint8_t get_id();
```
las funciones en snake_case, revisa todo y cambia las que hagan falta